app/src/routes/full-page-examples.test.mjs

```diff
@@ -336,21 +336,21 @@ describe(`http://localhost:${ports.app}/full-page-examples/`, () => {
         const body = await response.text()
         const $ = load(body)
         // Check the results are correct
-        expect($.html()).toContain('121,842 results')
+        expect($.html()).toContain('482,211 results')
```

Comment:
**question**: Why have the numbers in the tests changed? Is it due to the change in query parser?

Reply:
We pass the the query (as a string) into [`shuffle-seed`](https://www.npmjs.com/package/shuffle-seed)

So yeah it's gone from `URLSearchParams.prototype.toString()` to JSON which changes all the numbers

Same query name/values still